tools/generator/AnDOGenerator.py

```diff
@@ -12,13 +12,13 @@
 except ImportError:
     HAVE_PANDAS = False
 from ando.AnDOChecker import build_rule_regexp
-from ando.rulesStructured import rules_set
+from ando.rulesStructured import RULES_SET
 from ando.rulesStructured import DATA_EXTENSIONS
 from ando.rulesStructured import METADATA_EXTENSIONS
 
 MANDATORY_SUBFOLDER = "ephys"
```

Comment:
Could this be extracted from the RULES_SET?

Reply:
You mean we should extract the DATA and METADATAEXTENSIONS from the rules itself? I think in principle we could. I have to think about it a bit...

Reply:
no, my comment was actually on another line which is the following hardcoded definition:

'''
MANDATORY_SUBFOLDER = "ephys"
'''

Reply:
Ah, I see. Yes, that should be easier.

Reply:
This variable seems not to be used anyway. I removed it.